backend/src/impl/db_utils/benchmark_db_utils.py

```diff
@@ -255,11 +250,11 @@ def generate_dataframe_from_sys_infos(
         df_input["score"] = []
 
         # --- Create the actual data
-        for sys_name, sys_infos in system_dataset_results.items():
-            for dataset_config, dataset_metadata, sys_info_tmp in zip(
-                dataset_configs, dataset_metadatas, sys_infos
+        for sys_name, systems in system_dataset_results.items():
+            for dataset_config, dataset_metadata, system_tmp in zip(
+                dataset_configs, dataset_metadatas, systems
             ):
-                sys_info = unwrap(sys_info_tmp)
+                sys = unwrap(system_tmp)
```

Comment:
According to the type annotation, it seems that this `unwrap` is not necessary.

Reply:
Thanks!